preload.js

```diff
@@ -497,7 +504,7 @@ window.SMALL_GROUP_SIZE_LIMIT = 10;
 
 window.lokiFeatureFlags = {
   multiDeviceUnpairing: true,
-  privateGroupChats: false,
```

Comment:
Are we ready to enable this feature? I mean is everything ready elsewhere on the app and backend and other platforms for privateGroupChats?

Reply:
Yes. This is Kees recommendation.

Reply:
Why not just flip it to `true`?